dockerclient.go

```diff
@@ -258,6 +258,37 @@ func (client *DockerClient) ContainerChanges(id string) ([]*ContainerChanges, er
 	return changes, nil
 }
 
+func (client *DockerClient) ContainerStats(id string, stopChan <-chan struct{}) (<-chan StatsOrError, error) {
+	uri := fmt.Sprintf("/%s/containers/%s/stats", APIVersion, id)
```

Comment:
the general approach of this repo so far has just been to keep the API version the same despite adding API calls that rely on newer API versions (e.g. `ListNetworks`). Given that (1) we don't have a testing framework to actually test all of the dockerclient methods on docker engines of all versions and (2) it's not uncommon for API changes to be breaking and undocumented, switching the APIVersion has high risk and low reward :/

If we were to switch the APIVersion, it should probably be in a separate PR.